.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Make sure you add `.vscode` to you `.gitignore`

Reply:
Good catch, that snuck in there fixing a merge conflict